packages/react-aria-components/stories/index.stories.tsx

```diff
@@ -961,8 +965,10 @@ export const DropzoneWithRenderProps = (props) => (
   </div>
 );
 
+// TODO: Should data- attr pass to hidden input? It isn't duplicated, so not sure what the expectation is.
```

Comment:
I think it is fine that the data attributes make it onto the visually hidden input, don't wanna restrict what the user can pass their data attributes to.